core/src/main/scala/kafka/server/FetchSession.scala

```diff
@@ -759,7 +759,7 @@ class FetchManager(private val time: Time,
       cache.synchronized {
         cache.get(reqMetadata.sessionId()) match {
           case None => {
-            info(s"Created a new error FetchContext for session id ${reqMetadata.sessionId()}: " +
+            debug(s"Created a new error FetchContext for session id ${reqMetadata.sessionId()}: " +
               "no such session ID found.")
```

Comment:
Actually, can we clarify this log message a bit? What is a "error FetchContext"?

Reply:
An error fetch context is a type of FetchSessionContext which always sends back an error code.

However, it's probably better just to print out a message about a session error here, since the ErrorFetchContext is an implementation detail.